deployments/deploy-tests.ps1

```diff
@@ -1,10 +1,6 @@
-try {
+#try {
 
     #shows how to import modules
-    Import-Module .\powerform\pf-deployment-context\pf-deployment-context.psd1
-    Import-Module .\azure\resource-containers\pf-management-group\pf-management-group.psd1
-    Import-Module .\azure\resource-containers\pf-subscription\pf-subscription.psd1
-    Import-Module .\azure\resource-containers\pf-resource-group\pf-resource-group.psd1
     Import-Module .\azure\resources\pf-key-vault\pf-key-vault.psd1
     Import-Module .\azure\resources\pf-log-analytics\pf-log-analytics.psd1
```

Comment:
Lots of good stuff in this deploy-tests now.  Take a look through all these scenarios and let me know if you have any questions.